package.json

```diff
@@ -39,5 +39,8 @@
     "winston": "^3.11.0",
     "yaml": "^2.3.4",
     "yargs": "^17.7.2"
+  },
+  "peerDependencies": {
+    "picomatch": "^3.0.1"
```

Comment:
Not used?

Reply:
I ran into an error about `picomatch` when I ran one of the `npm run …` scripts, but now I can't reproduce this. Rolling it back.